core/src/main/scala/org/apache/spark/metrics/sink/StatsdSink.scala

```diff
@@ -32,12 +39,24 @@ private[spark] object StatsdSink {
   val STATSD_KEY_PERIOD = "period"
   val STATSD_KEY_UNIT = "unit"
   val STATSD_KEY_PREFIX = "prefix"
+  val STATSD_KEY_EXCLUDES = "excludes"
+  val STATSD_KEY_INCLUDES = "includes"
+  val STATSD_KEY_EXCLUDES_ATTRIBUTES = "excludesAttributes"
+  val STATSD_KEY_INCLUDES_ATTRIBUTES = "includesAttributes"
+  val STATSD_KEY_USE_REGEX_FILTERS = "useRegexFilters"
+  val STATSD_KEY_USE_SUBSTRING_MATCHING = "useSubstringMatching"
 
   val STATSD_DEFAULT_HOST = "127.0.0.1"
   val STATSD_DEFAULT_PORT = "8125"
   val STATSD_DEFAULT_PERIOD = "10"
   val STATSD_DEFAULT_UNIT = "SECONDS"
   val STATSD_DEFAULT_PREFIX = ""
+  val STATSD_DEFAULT_EXCLUDES = ""
+  val STATSD_DEFAULT_INCLUDES = ""
+  val STATSD_DEFAULT_EXCLUDES_ATTRIBUTES = ""
+  val STATSD_DEFAULT_INCLUDES_ATTRIBUTES = ""
+  val STATSD_DEFAULT_USE_REGEX_FILTERS = "false"
+  val STATSD_DEFAULT_USE_SUBSTRING_MATCHING = "false"
```

Comment:
I'd suggest assigning defaults of appropriate types instead of Strings. E.g. `Set.empty[String]` for includes/excludes, and `false` for booleans

Reply:
We cannot have types other than string because these variables are being passed to the `Properties.getProperty` and it accepts only string type.